lib/streamlit/elements/map.py

```diff
@@ -101,16 +120,70 @@ def map(
 
         Parameters
         ----------
-        data : pandas.DataFrame, pandas.Styler, pyarrow.Table, numpy.ndarray, pyspark.sql.DataFrame, snowflake.snowpark.dataframe.DataFrame, snowflake.snowpark.table.Table, Iterable, dict, or None
-            The data to be plotted. Must have two columns:
-
-            - latitude called 'lat', 'latitude', 'LAT', 'LATITUDE'
-            - longitude called 'lon', 'longitude', 'LON', 'LONGITUDE'.
+        data : pandas.DataFrame, pandas.Styler, pyarrow.Table, numpy.ndarray,
+            pyspark.sql.DataFrame, snowflake.snowpark.dataframe.DataFrame,
+            snowflake.snowpark.table.Table, Iterable, dict, or None
+            The data to be plotted.
+
+        latitude : str or None
+            The name of the column containing the latitude coordinates of
+            the datapoints in the chart. This argument can only be supplied
+            by keyword.
+
+            If None, the latitude data will come from any column named 'lat',
+            'latitude', 'LAT', or 'LATITUDE'.
+
+        longitude : str or None
+            The name of the column containing the latitude coordinates of
+            the datapoints in the chart. This argument can only be supplied
+            by keyword.
+
+            If None, the latitude data will come from any column named 'lon',
+            'longitude', 'LON', or 'LONGITUDE'.
+
+        color : str or tuple
+            The color of the circles representing each datapoint. This argument
+            can only be supplied by keyword.
+
+            Can be:
+            - None, to use the default color.
+            - A hex string like "#ffaa00" or "#ffaa0088".
+            - An RGB or RGBA tuple with the red, green, blue, and alpha
+              components specified as ints from 0 to 255 or floats from 0.0 to
+              1.0.
+            - The name of the column to use for the color. Cells in this column
+              should contain colors represented as a hex string or color tuple,
+              as described above.
+
+            If passing in a str, the Matplotlib library must be installed.
+
+        size : str, float, or None
+            The size of the circles representing each point. This argument can
```

Comment:
From reading this, it's not super intuitive what a number 100 represents within the plot. Is there any information that we could give here that gives intuition of how big 100 will be out of curiosity?

Reply:
I had no idea, but apparently this is "meters" by default. DeckGL also supports "pixels", which means the datapoints don't scale when you zoom in/out.

So a potential improvement later would be to support pixel sizes.

Reply:
Added this info to the docs.